crates/adapters/src/format/avro/schema.rs

```diff
@@ -475,6 +476,7 @@ impl AvroSchemaBuilder {
                     attributes: BTreeMap::new(),
                 })
             }
+            SqlType::Uuid => AvroSchema::String,
```

Comment:
There seems to also be a `Bytes` type in the `AvroSchema`, is there a particular reason for Parquet a binary representation is used whereas for Avro a string?